docs/source/routing/query-planning/query-planning-best-practices.mdx

```diff
@@ -0,0 +1,181 @@
+---
+title: Query Planning Best Practices
+subtitle: What things should we keep in mind as we construct our supergraph and the operations that consume them?
+---
+
+When working with Apollo Federation, changes in our schema can have unexpected impact on the complexity of our graph.
+While many issues can be found at build time with [schema composition](https://www.apollographql.com/docs/graphos/schema-design/federated-schemas/composition), others changes may lead to issues that only arise at runtime while generating or executing query plans.
```

Comment:
I believe links can be relative here, and don't need the full domain, although it's not going to harm anything here.

Reply:
Also, it may be worth expounding on what issues may arise- to my knowledge, composition should be catching them all and us flagging that QP may have issues is not possibly correct here.